platform-includes/user-feedback/install/javascript.mdx

```diff
@@ -1,17 +1,28 @@
-The User Feedback integration is **already included** in your browser or framework SDK NPM packages. If you're using CDN bundles instead of NPM packages, you need to load the User Feedback integration CDN bundle in addition to your browser bundle:
+The User Feedback integration is **already included** in your NPM packages. You don't need to install anything else to use it.
 
-```bash {tabTitle:npm}
-npm install @sentry/browser --save
-```
+If you're using the Loader Script, you can lazy load the User Feedback integration like this:
 
-```bash {tabTitle:yarn}
-yarn add @sentry/browser
-```
+```javascript {tabTitle: Loader}
+window.sentryOnLoad = function () {
+  Sentry.init({
+    // add other configuration here
+  });
 
-```bash {tabTitle:pnpm}
-pnpm add @sentry/browser
+  Sentry.lazyLoadIntegration("feedbackIntegration")
+    .then((feedbackIntegration) => {
+      Sentry.addIntegration(feedbackIntegration({
+      	// User Feedback configuration options
+      }));
+    })
+    .catch(() => {
+      // this can happen if e.g. a network error occurs,
+      // in this case User Feedback will not be enabled
+    });
+};
 ```
 
+If you're using CDN bundles instead of NPM packages, you need to load the User Feedback integration CDN bundle in addition to your browser bundle:
```

Comment:
"in addition" seems like it should go away. People should pick one CDN script with the combination of integrations that they want to have.

Reply:
yeah, good point, we call this out in the snippet anyhow!